### ISSUE TITLE
Text mode is the only way to accelerate the training process？

### ISSUE THREAD

**Issue by ouyangmeizi** [yanpanlau/DDPG-Keras-Torcs#55]
Hi,everyone！
Whether text mode is the only way to accelerate the training process？ I need to get picture from observation as input to train the model, so I can not use the text mode. But the training process is so slow, does anybody know how to accelerate the training process？
Thanks！

**Comment by DavisZuo** [comment]
Have u tried to terminate episodes when the car moves out or stay still for a long time? This may helps to accelerate.

**Comment by BCWang93** [comment]
> Have u tried to terminate episodes when the car moves out or stay still for a long time? This may helps to accelerate.

hello,your means is to turn off the terminate,when its very slow?

**Comment by DavisZuo** [comment]
yes, terminate the current training episode when it stays too long. 408780378 邮箱408780378@qq.com 签名由 网易邮箱大师 定制 On 03/13/2019 10:52, Bcw93 wrote: Have u tried to terminate episodes when the car moves out or stay still for a long time? This may helps to accelerate. hello,your means is to turn off the terminate,when its very slow? — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by BCWang93** [comment]
> yes, terminate the current training episode when it stays too long. 408780378 邮箱408780378@qq.com 签名由 网易邮箱大师 定制 On 03/13/2019 10:52, Bcw93 wrote: Have u tried to terminate episodes when the car moves out or stay still for a long time? This may helps to accelerate. hello,your means is to turn off the terminate,when its very slow? — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

Hi,do you know how to use the text model?I don't want the gui display in training.Thank you very much!

**Comment by Aarnnity** [comment]
> Hi,everyone！
> Whether text mode is the only way to accelerate the training process？ I need to get picture from observation as input to train the model, so I can not use the text mode. But the training process is so slow, does anybody know how to accelerate the training process？
> Thanks！

Excuse me, can you successfully obtain the current image? I set vision=True, an error occurred, KeyError:'img'. Have you encountered this situation?

**Comment by Aarnnity** [comment]
Excuse me, can you successfully obtain the current image? I set vision=True, an error occurred, KeyError:'img'. Have you encountered this situation?
